sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/ImportCertificateOptions.java

```diff
@@ -47,12 +47,12 @@
     /**
      * Creates instance of CertificateImportOptions.
      * @param name The name of the key.
-     * @param value The PFX or PEM formatted value of the certificate containing both the x509 certificates and the private key.
+     * @param certificate The PFX or PEM formatted value of the certificate containing both the x509 certificates and the private key.
      */
-    public ImportCertificateOptions(String name, byte[] value) {
-        Objects.requireNonNull(value, "The certificate value parameter cannot be null.");
+    public ImportCertificateOptions(String name, byte[] certificate) {
+        Objects.requireNonNull(certificate, "The certificate value parameter cannot be null.");
```

Comment:
nit The certificate ~~value~~ parameter cannot be null.

Reply:
fixed.